charts/linkerd2/templates/controller.yaml

```diff
@@ -91,6 +110,38 @@ spec:
         volumeMounts:
         - mountPath: /var/run/linkerd/config
           name: config
+      - args:
```

Comment:
I think we should also update line 84 in this file to point the public-api at the new linkerd-dst service